src/terraform/commands.go

```diff
@@ -3,14 +3,30 @@ package terraform
 import (
 	"fmt"
 
+	"github.com/Originate/exosphere/src/docker/tools"
 	"github.com/Originate/exosphere/src/types"
 	"github.com/Originate/exosphere/src/util"
 )
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	homeDir, err := util.GetHomeDirectory()
+	if err != nil {
+		return err
+	}
+	volumes := []string{
+		fmt.Sprintf("%s:/app", deployConfig.TerraformDir),
+		fmt.Sprintf("%s/.aws:/root/.aws", homeDir),
+	}
 	backendConfig := fmt.Sprintf("-backend-config=profile=%s", deployConfig.AwsConfig.Profile)
-	return util.RunAndPipe(deployConfig.TerraformDir, []string{}, deployConfig.Writer, "terraform", "init", "-force-copy", backendConfig)
+	dockerRunConfig := tools.RunConfig{
+		Volumes:    volumes,
+		WorkingDir: "/app",
+		ImageName:  fmt.Sprintf("hashicorp/terraform:%s", deployConfig.TerraformVersion),
```

Comment:
Thoughts on making the terraform version and terraform image name constants in the terraform package? The module hash could moved in there as well.